### ISSUE TITLE
macos mojave telepresence install bug

### ISSUE THREAD

**Issue by mostafaD** [telepresenceio/telepresence#1076]
### What were you trying to do?

telepresence --new-deployment mostafa22-debug --run-shell --env-json dev_cluster_env2.json

### What did you expect to happen?
connect to telepresence 

### What happened instead?
the process failed with the following logs and asked me to report a bug

use https://gist.github.com to pass along full telepresence.log)

### Automatically included information

Command line: `['/usr/local/bin/telepresence', '--new-deployment', 'mostafa22-debug', '--run-shell', '--env-json', 'dev_cluster_env2.json']`
Version: `0.99`
Python version: `3.5.2 (v3.5.2:4def2a2901a5, Jun 26 2016, 10:47:25) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)]`
kubectl version: `Client Version: v1.10.7 // Server Version: v1.13.6-gke.13`
oc version: `(error: [Errno 2] No such file or directory: 'oc')`
OS: `Darwin Mostafa.local 18.0.0 Darwin Kernel Version 18.0.0: Wed Aug 22 20:13:40 PDT 2018; root:xnu-4903.201.2~1/RELEASE_X86_64 x86_64`

```
Traceback (most recent call last):
  File "/usr/local/bin/telepresence/telepresence/cli.py", line 136, in crash_reporting
    yield
  File "/usr/local/bin/telepresence/telepresence/main.py", line 60, in main
    remote_info = start_proxy(runner)
  File "/usr/local/bin/telepresence/telepresence/proxy/__init__.py", line 95, in start_proxy
    run_id=run_id,
  File "/usr/local/bin/telepresence/telepresence/proxy/remote.py", line 202, in get_remote_info
    wait_for_pod(runner, remote_info)
  File "/usr/local/bin/telepresence/telepresence/proxy/remote.py", line 134, in wait_for_pod
    "Pod isn't starting or can't be found: {}".format(pod["status"])
RuntimeError: Pod isn't starting or can't be found: {'startTime': '2019-07-15T12:46:31Z', 'qosClass': 'Burstable', 'containerStatuses': [{'state': {'waiting': {'reason': 'CrashLoopBackOff', 'message': 'Back-off 1m20s restarting failed container=mostafa22-debug pod=mostafa22-debug-86bb78c4c7-sqdkh_default(9233a0dd-a6fe-11e9-b291-42010a8e010e)'}}, 'containerID': 'docker://8548fcbdc7370346b4261053ca3ed91bf22d333b53abf03833f17c7be7e94c49', 'imageID': 'docker-pullable://datawire/telepresence-k8s@sha256:f9b640fa6640a0437cbafb0a3a238b10daf86e117e5a3ca4d4cd4b07eee76f9c', 'ready': False, 'image': 'datawire/telepresence-k8s:0.99', 'name': 'mostafa22-debug', 'lastState': {'terminated': {'reason': 'Error', 'startedAt': '2019-07-15T12:47:56Z', 'exitCode': 1, 'containerID': 'docker://8548fcbdc7370346b4261053ca3ed91bf22d333b53abf03833f17c7be7e94c49', 'finishedAt': '2019-07-15T12:47:56Z'}}, 'restartCount': 4}], 'podIP': '10.20.25.96', 'phase': 'Running', 'hostIP': '10.142.0.37', 'conditions': [{'type': 'Initialized', 'lastTransitionTime': '2019-07-15T12:46:31Z', 'lastProbeTime': None, 'status': 'True'}, {'reason': 'ContainersNotReady', 'lastProbeTime': None, 'status': 'False', 'message': 'containers with unready status: [mostafa22-debug]', 'lastTransitionTime': '2019-07-15T12:46:31Z', 'type': 'Ready'}, {'reason': 'ContainersNotReady', 'lastProbeTime': None, 'status': 'False', 'message': 'containers with unready status: [mostafa22-debug]', 'lastTransitionTime': '2019-07-15T12:46:31Z', 'type': 'ContainersReady'}, {'type': 'PodScheduled', 'lastTransitionTime': '2019-07-15T12:46:31Z', 'lastProbeTime': None, 'status': 'True'}]}

```

Logs:

```
[81] Capturing: kubectl --context gke_dev-env-203117_us-east1-b_dev --namespace default get pod mostafa22-debug-86bb78c4c7-sqdkh -o json
 127.4 TEL | [81] captured in 1.39 secs.
 127.6 TEL | [82] Capturing: kubectl --context gke_dev-env-203117_us-east1-b_dev --namespace default get pod mostafa22-debug-86bb78c4c7-sqdkh -o json
 128.9 TEL | [82] captured in 1.30 secs.
 129.2 TEL | [83] Capturing: kubectl --context gke_dev-env-203117_us-east1-b_dev --namespace default get pod mostafa22-debug-86bb78c4c7-sqdkh -o json
 130.5 TEL | [83] captured in 1.29 secs.
 130.7 TEL | [84] Capturing: kubectl --context gke_dev-env-203117_us-east1-b_dev --namespace default get pod mostafa22-debug-86bb78c4c7-sqdkh -o json
 132.0 TEL | [84] captured in 1.22 secs.
 132.2 TEL | [85] Capturing: kubectl --context gke_dev-env-203117_us-east1-b_d

**Comment by ark3** [comment]
Sorry about the crash. Can you pass along the output of `kubectl logs` from that pod? In this case that would be

    kubectl --context gke_dev-env-203117_us-east1-b_dev --namespace default logs mostafa22-debug-86bb78c4c7-sqdkh

Thank you.

**Comment by mostafaD** [comment]
Hi

i tried to regenerate the error again but it didn’t happen and the process succeeded so i think it is randomly working 
i forward you if i faced this error again

thanks

**Comment by mostafaD** [comment]
hi i have faced the connection problem again now
here is the logs

Mostafa:~ mostafa$ kubectl logs mostafa23-debug-7d68777bd-xffn6
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@         WARNING: UNPROTECTED PRIVATE KEY FILE!          @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
Permissions 0640 for '/etc/ssh/ssh_host_rsa_key' are too open.
It is required that your private key files are NOT accessible by others.
This private key will be ignored.
key_load_private: bad permissions
Could not load host key: /etc/ssh/ssh_host_rsa_key
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@         WARNING: UNPROTECTED PRIVATE KEY FILE!          @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
Permissions 0640 for '/etc/ssh/ssh_host_dsa_key' are too open.
It is required that your private key files are NOT accessible by others.
This private key will be ignored.
key_load_private: bad permissions
Could not load host key: /etc/ssh/ssh_host_dsa_key
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@         WARNING: UNPROTECTED PRIVATE KEY FILE!          @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
Permissions 0640 for '/etc/ssh/ssh_host_ecdsa_key' are too open.
It is required that your private key files are NOT accessible by others.
This private key will be ignored.
key_load_private: bad permissions
Could not load host key: /etc/ssh/ssh_host_ecdsa_key
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@         WARNING: UNPROTECTED PRIVATE KEY FILE!          @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
Permissions 0640 for '/etc/ssh/ssh_host_ed25519_key' are too open.
It is required that your private key files are NOT accessible by others.
This private key will be ignored.
key_load_private: bad permissions
Could not load host key: /etc/ssh/ssh_host_ed25519_key
sshd: no hostkeys available -- exiting.

thanks

**Comment by ark3** [comment]
Thank you. This is #1013, which was fixed in release 0.100. Can you please upgrade and try again?

**Comment by donnyyung** [comment]
I believe this is no longer an issue in Telepresence 2. Here are the docs on how to install Telepresence (https://www.telepresence.io/docs/latest/install/), please re-open if you still see this issue in our latest version!
